examples/ConfluentCloud/Program.cs

```diff
@@ -86,13 +86,13 @@ static void Main(string[] args)
                 AutoOffsetReset = AutoOffsetResetType.Earliest
             };
 
-            using (var consumer = new Consumer<Null, string>(cConfig))
-            { 
+            using (var consumer = new Consumer(cConfig))
+            {
                 consumer.Subscribe("dotnet-test-topic");
 
                 try
                 {
-                    var consumeResult = consumer.Consume();
+                    var consumeResult = consumer.Consume<Null, string>();

```

Comment:
note all the automatic type inferring and executing of relevant serializers going on here!